README.md

```diff
@@ -262,7 +263,7 @@ h('a', {class: {active: true, selected: false}}, 'Toggle');
 Allows you to set properties on DOM elements.
 
 ```javascript
-h('a', {props: {href: '/foo'}, 'Go to Foo');
+h('a', { props: {href: '/foo'} }, 'Go to Foo');
```

Comment:
I don't put spaces there. Actually they should be removed in the section on the attributes module.